lib/fastlane/plugin/ionic/actions/ionic_action.rb

```diff
@@ -238,6 +244,16 @@ def self.available_options
             is_string: true,
             default_value: ''
           ),
+          FastlaneCore::ConfigItem.new(
+            key: :package_type,
```

Comment:
Thoughts on this rename? We have `type` and `package_type` and that seems like it could get a little confusing. I think I'd like to rename this to 'android_package_type` and then eventually deprecate `type` and rename it to `ios_package_type`. 

```suggestion
            key: :android_package_type,
```